users/views.py

```diff
@@ -9,7 +9,58 @@
 from users.validators import UserValidator
 from my_settings      import SECRET_KEY, ALGORITHM
 
+class SignupView(View):
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            name          = data['name']
+            phone         = data['phone']
+            email         = data['email']
+            date_of_birth = data['date_of_birth']
+            password      = data['password']
+            gender        = data['gender']
+
+            V = UserValidator()
+            V.validate_name(name)
+            V.validate_phone(phone)
+            V.validate_email(email)
+            V.validate_date_of_birth(date_of_birth)
+            V.validate_password(password)
+            V.validate_gender(gender)
```

Comment:
변수 명은 줄여서 쓰지 않도록 주의해주세요~
```suggestion
            user_validator = UserValidator()
            user_validator.validate_name(name)
            user_validator.validate_phone(phone)
            user_validator.validate_email(email)
            user_validator.validate_date_of_birth(date_of_birth)
```

Reply:
넵 바로 수정하겠습니다!